src/content/docs/en/core-concepts/astro-components.mdx

```diff
@@ -291,7 +291,7 @@ Named slots can also be passed to [UI framework components](/en/core-concepts/fr
 
 
 :::note
-An astro slot name can not be dynamically generated, such as within a map function. If this feature is needed within UI framework components, it might be best to generate these dynamic slots within the framework itself.
+Dynamically generating an astro slot name, such as within a map function, was added in Astro v4.2.0. For earlier versions of Astro, it might be best to generate these dynamic slots within the framework itself.
```

Comment:
```suggestion
Dynamically generating an Astro slot name, such as within a map function, was added in Astro v4.2.0. For earlier versions of Astro, it might be best to generate these dynamic slots within the framework itself.
```